release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2458,6 +2460,25 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=18786
 
 * Following an {product-title} single node reboot, all pods are restarted which causes significant load and longer than normal pod creation times. This happens because the Container Network Interface (CNI) is not able to process the `pod add` events quickly enough. The following error message is displayed: `timed out waiting for OVS port binding`. The {product-title} single node instance eventually recovers, though more slowly than expected. This known issue applies to the {product-title} version 4.8.15 and later. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986216[*BZ#1986216*])
 
+* Prior to {product-title} 4.8 the default balancing algorithm was `leastconn`. The default was changed to `random` in {product-title} 4.8.0 for non-passthrough routes. Switching to `random` increases memory consumption for every HAProxy process, and the cumulative memory usage is significant if you have several websocket connections. To mitigate this significant memory consumption, the default balancing algorithm was reverted to `leastconn` in {product-title} 4.8. Once there is a solution that does not incur significant memory requirements the default will be changed to `random` in a future {product-title} release.
```

Comment:
```suggestion
* Prior to {product-title} 4.8 the default load balancing algorithm was `leastconn`. The default was changed to `random` in {product-title} 4.8.0 for non-passthrough routes. Switching to `random` is incompatible with environments that need to use long-running websocket connections because it significantly increases memory consumption in those environments.  To mitigate this significant memory consumption, the default load balancing algorithm was reverted to `leastconn` in {product-title} 4.8. Once there is a solution that does not incur significant memory usage, the default will be changed to `random` in a future {product-title} release.
```

Reply:
Updated, thank you!